samples/quantum-approximation-optimization/qaoa-for-quadratic-unconstrained-binary-optimization.ipynb

```diff
@@ -0,0 +1,823 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# QAOA and QAA to solve a QUBO problem"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "metadata": {},
+   "source": [
+    "# Restart the kernel once you have run this cell\n",
+    "!pip install -U -q azure-quantum\n",
+    "\n",
+    "try:\n",
+    "    import pulser\n",
+    "    import pulser_simulation\n",
+    "except ImportError:\n",
+    "    # This installs two python packages:\n",
+    "    #   - pulser (core package)\n",
+    "    #   - pulser_simulation\n",
+    "    # If you don't need simulation in your own workflow, you may run instead:\n",
+    "    #   !pip -q install pulser-core\n",
+    "    # https://pulser.readthedocs.io/en/stable/installation.html\n",
+    "    !pip -q install --upgrade pulser"
+   ],
```

Comment:
Could we now remove this cell since we implemented `pip install azure-quantum[pulser]` and the Pulser will be installed automatically in Notebooks?

Reply:
Yes sorry I briefly thought about it then forgot. Cell removed.